salt/modules/vmctl.py

```diff
@@ -0,0 +1,378 @@
+# -*- coding: utf-8 -*-
+# $OpenBSD$
+'''
+Manage vms running on the OpenBSD VMM hypervisor using vmctl(8).
+
```

Comment:
A versionadded for `Fluorine` will be needed for this new file.

Reply:
Check, I'll it.